### PR TITLE
feat: add non-profiled consumption per grid area

### PR DESCRIPTION
**Pull request by johevemi**
## References

* #819

### CODE REVIEW

File: source/databricks/package/balance_fixing.py [hunk 1]
```diff
@@ -124,6 +124,17 @@ def _calculate_non_profiled_consumption(
...
+    # Non-profiled consumption per grid area
+    consumption_per_ga = agg_steps.aggregate_non_profiled_consumption_ga(
+        consumption_per_ga_and_brp_and_es
+    )
+
+    result_writer.write_per_ga(
+        consumption_per_ga,
+        TimeSeriesType.NON_PROFILED_CONSUMPTION,
+        Grouping.total_ga,
+    )
```
[the remark this answers is not in the document]
Comment (johevemi) [reply source/databricks/package/balance_fixing.py:136]: This is the last calculation for Non-profiled consumption, so the _calculate_non_profiled_consumption will not be expanded further. If this is not what you had in mind please reach out.

**Comment by johevemi** [comment]
> No tests?

Test has been added to test_calculator_job. I refactored a test that used a for loop to a parametrized test

File: source/databricks/package/balance_fixing.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (BjarkeMeier) [reply source/databricks/package/balance_fixing.py:136]: Ah, ok 😄

**Comment by BjarkeMeier** [comment]
> > No tests?
> 
> Test has been added to test_calculator_job. I refactored a test that used a for loop to a parametrized test

Nice 👍 Approved.

**Pull request merged**